modules/smartyadsBidAdapter.js

```diff
@@ -26,6 +31,25 @@ function isBidResponseValid(bid) {
   }
 }
 
+function getAdUrlByRegion(bid) {
+  let adUrl;
+
+  if (bid.params.region && adUrls[bid.params.region]) {
+    adUrl = adUrls[bid.params.region];
+  } else {
+    try {
+      const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+      const region = timezone.split('/')[0];
+      if (region === 'Europe') adUrl = adUrls['EU'];
```

Comment:
Just to be sure, you don't want to check about SPG?

Reply:
@osazos yes, we want to test only EU and US_EAST regions now